pkg/tfbridge/schema.go

```diff
@@ -1228,6 +1258,7 @@ func MakeTerraformConfig(ctx context.Context, p *Provider, m resource.PropertyMa
 	return MakeTerraformConfigFromInputs(ctx, p.tf, inputs), cctx.Assets, nil
 }
 
+// TODO: this is now unused internally, can we delete it?
```

Comment:
This would technically be a breaking change, are we fine to delete this or should we keep it?

Reply:
I'm ok removing it. We've made breaking changes to this function recently (introducing `ctx`) and any consumers can easily inline the function.

This is one of those functions that really shouldn't have been public to begin with.